ibm/mas_devops/roles/gencfg_jdbc/README.md

```diff
@@ -3,51 +3,97 @@ gencfg_jdbc
 
-Role Variables
---------------
-
-### mas_instance_id
-Providing this and `mas_config_dir` will instruct the role to generate a JdbcCfg template that can be used to configure MAS to connect to this database.
-
-- Environment Variable: `MAS_INSTANCE_ID`
-- Default: None
-- 
-### mas_workspace_id
-This is only used when both `mas_config_dir` and `mas_instance_id` are set, and `mas_config_scope` is set to either `ws` or `wsapp`
-
-- Environment Variable: `MAS_WORKSPACE_ID`
-- Default: None
-
+Role Variables - Data Source
+-------------------------------------------------------------------------------
 ### db_instance_id
 Defines the instance id that is used for the db configure in MAS installation
 
+- **Required**
 - Environment Variable: `DB_INSTANCE_ID`
 - Default: `dbinst`
 
 ### db_username
 Defines the username that is used for the db configure in MAS installation
 
+- **Required**
 - Environment Variable: `MAS_JDBC_USER`
 - Default: None
-- 
+
 ### jdbc_instance_password
 Defines the password that is used to connect to db in MAS installation
 
+- **Required**
 - Environment Variable: `MAS_JDBC_PASSWORD`
 - Default: None
-- 
+
 ### jdbc_url
-Defines the jdbc url  that is used to connect to db in MAS installation , Append ;sslConnection=true to the URL so that it has the form jdbc:db2://hostname:port/database_name:sslConnection=true .
+Defines the jdbc URL that is used to connect to db in MAS installation:
 
+- **Required**
 - Environment Variable: `MAS_JDBC_URL`
 - Default: None
-- 
+
+!!! tip
+    Example URL strings:
+
+    - IBM Db2 (insecure): `jdbc:db2://dbserverxx:50000/maxdbxx`
+    - IBM Db2 (secure): `jdbc:db2://dbserverxx:50000/maxdbxx:sslConnection=true`
+    - Oracle Database: `jdbc:oracle:thin:@dbserverxx:1521:maximo`
+    - Microsoft SQL Server (insecure): `jdbc:sqlserver://;serverName=dbserverxx;portNumber=1433;databaseName=msdbxx;integratedSecurity=false;sendStringParametersAsUnicode=false;selectMethod=cursor;encrypt=false;trustServerCertificate=false;`
+    - Microsoft SQL Server (secure): `jdbc:sqlserver://;serverName=dbserverxx;portNumber=1433;databaseName=msdbxx;integratedSecurity=false;sendStringParametersAsUnicode=false;selectMethod=cursor;encrypt=true;trustServerCertificate=true;`
+
 ### db_pem_file
 Defines the location of the pem file used for JDBC connection in MAS installation
 
+- Optional
 - Environment Variable: `MAS_JDBC_CERT_LOCAL_FILE`
 - Default: None
 
+
+Role Variables - MAS Configuration
+-------------------------------------------------------------------------------
+### mas_config_scope
+Configure whether to generate a binding suitable for System, Workspace, Application, or Workspace-Application use within MAS (`system`, `ws`, `app`, or `wsapp`).
+
+- **Required**
+- Environment Variable: `MAS_CONFIG_SCOPE`
+- Default: None
```

Comment:
Default here should be `system`

Reply:
There's no default in the role:

```
# MAS instance configuration
mas_config_scope: "{{ lookup('env', 'MAS_CONFIG_SCOPE')}}"
```

The default is only applied at the playbook level:
```
    # Manage components and server bundle configurations
    mas_config_scope: "{{ lookup('env', 'MAS_CONFIG_SCOPE') | default('system', true) }}" # Supported values are "system", "ws", "app", or "wsapp"
    mas_appws_bindings_jdbc: "{{ lookup('env', 'MAS_APPWS_BINDINGS_JDBC') | default('system', true) }}"
```